### ISSUE TITLE
Header Bar Changes

### ISSUE THREAD

**Issue by Lynx007** [laiglinne-ff/Project_TheStone#9]
Not really an issue more of a suggestion and if it's the wrong place to post it i apologise,

As it is now we have a Header Bar that looks like this

![image](https://user-images.githubusercontent.com/34261791/36368969-a6122df4-15ad-11e8-8f52-3e6896b9a85c.png)

would it be possible to get one that looks more like this

![image](https://user-images.githubusercontent.com/34261791/36369001-c6dc85d4-15ad-11e8-97bb-adb422aa2cba.png)

basically half the height and everything resized and put onto one line, just for a more streamlined sleek look without a huge header bar!

Thanks!
